src/Validation/Validation.php

```diff
@@ -1431,6 +1431,17 @@ public static function isArray($value)
     }
 
     /**
+     * Check that the input value is a 6 digits hex color.
```

Comment:
I'm fine for a specific validation for a specific context, but in that case the context should be reflected in the method name .

Alternatively, there could be two methods:

1. a general purpose method, providing @ionas proposed flag parameter
2. a method specific to the input field, which in turn calls the general purpose method with the flag parameter set